torchvision/datasets/voc.py

```diff
@@ -202,6 +202,12 @@ def __getitem__(self, index: int) -> Tuple[Any, Any]:
         Returns:
             tuple: (image, target) where target is a dictionary of the XML tree.
         """
+        # If defusedxml exists, use defusedxml.ElementTree instead.
+        # Else, use xml.ElementTree as normal.
+        try:
+            import defusedxml.ElementTree as ET
+        except ImportError:
+            pass
```

Comment:
Can you please move this to the main import block, to do something like:
```
try:
  import defuesxml.ElementTree as ET
except ImportError:
  import xml.ElementTree as ET
```

Reply:
Thank you @malfet , I've made the changes to import ` xml.etree.ElementTree.Element as ET_Element` and `defusedxml.ElementTree.parse as ET_parse` separately.